src/server/game/Garrison/GarrisonMgr.cpp

```diff
@@ -461,3 +462,59 @@ void GarrisonMgr::LoadFollowerClassSpecAbilities()
 
     TC_LOG_INFO("server.loading", ">> Loaded %u garrison follower class spec abilities in %u.", count, GetMSTimeDiffToNow(msTime));
 }
+
+GarrisonTalentNPC const* GarrisonMgr::GetTalentNPCEntry(int32 entry) const
+{
+    return Trinity::Containers::MapGetValuePtr(_garrisonTalentNPCs, entry);
+}
+
+void GarrisonMgr::LoadTalentNPCs()
+{
+    //                                                      0                 1                    2
+    QueryResult result = WorldDatabase.Query("SELECT npcEntry, garrTalentTreeID, friendshipFactionID FROM garrison_talent_npc");
```

Comment:
you defined the fields in UpperCamelCase in sql file, you should use naming 1:1 like it is there